src/CHIPBackend.hh

```diff
@@ -537,7 +537,14 @@ public:
     }
   }
   void trackImpl();
-  void track() { std::call_once(TrackCalled, &CHIPEvent::trackImpl, this); }
+  void track() {
+    std::lock_guard<std::mutex> Lock(Mtx);
+    //std::call_once(TrackCalled, &CHIPEvent::trackImpl, this);
+    if (!TrackCalled) {
+      trackImpl();
+      TrackCalled = true;
+    }
+  }
```

Comment:
Right, this change was necessary when we moved to reusing events. Probably what was happening was that events would get recycled and not get tracked due to `call_once` protection